### PR TITLE
A10 ssrf

### PR DESCRIPTION
**Pull request by marco-ippolito**
slides for [ssrf](https://owasp.org/Top10/A10_2021-Server-Side_Request_Forgery_%28SSRF%29/)

### CODE REVIEW

File: slides.md [hunk 1]
```diff
@@ -1053,10 +1053,87 @@ export default function profile(fastify) {
...
+
+</div>
+
+---
+
+# A10: Server Side Request Forgery: The Attack
+
+<div class="dense">
+
+- Run the server for step 5 (`cd src/a10-server-side-request-forgery`, `npm start`)
```
Comment (groozin) [inline slides.md:1087]:
```suggestion
- Run the server for step 10 (`cd src/a10-server-side-request-forgery`, `npm start`)
```

Comment (groozin) [reply slides.md:1087]: This is slide 10 right? Or do we mean here some other kinds of steps?

File: slides.md [hunk 2]
```diff
@@ -1053,10 +1053,87 @@ export default function profile(fastify) {
...
+
+</div>
+
+---
+
+# A10: Server Side Request Forgery: How to fix
+
+<div class="dense">
+
+- Sanitized the url making sure it's valid
```
Comment (groozin) [inline slides.md:1100]:
```suggestion
- Sanitize the url making sure it's valid
```

File: slides.md [hunk 3]
```diff
@@ -1053,10 +1053,87 @@ export default function profile(fastify) {
...
+- Create a whitelist of allowed domains by adding them in the database column `allowedImageDomain`
+- Make sure the requested domain it's in the whitelist
```
Comment (groozin) [inline slides.md:1102]:
```suggestion
- Make sure the requested domain is in the whitelist
```

**Review by washingtonsoares — DISMISSED** [review]

File: slides.md (hunk 1 above)
Comment (marco-ippolito) [reply slides.md:1087]: yes this is a typo

**Pull request merged**